docs/src/guide/topology.md

```diff
@@ -129,10 +129,10 @@ ssh -i /path/to/private_key -J <username>@bastion_ip opc@worker_node_private_ip
 ****
 N.B. If your private ssh key has a different name or path than the default "~/.ssh/id_*" e.g "~/.ssh/dev_rsa", you will need to add the private key to your ssh agent:
 
-----
+```
```

Comment:
Please add this in the instructions for accessing the operator as well

Reply:
WIP